Services/DataX.Config/DataX.Config/ConfigGeneration/Processor/S500_ResolveOutputs.cs

```diff
@@ -175,6 +175,13 @@ private async Task<FlowOutputSpec[]> ProcessOutputs(FlowGuiOutput[] uiOutputs, R
                                 flowOutput.BlobOutput = blobOutput;
                                 break;
                             }
+                        case "sqlserver":
+                            {
+                                var sqlOutput =  await ProcessOutputSql(configName, output);
+                                Ensure.EnsureNullElseThrowNotSupported(flowOutput.SqlOutput, "Multiple target Sql ouptut for same dataset not supported.");
```

Comment:
output typo.  1 SQL output for a Spark SQL table?

Reply:
One spark dataset can output to only one output (any). Different table is also not supported.

Reply:
ok.  typo in ouptut.